### PR TITLE
copy mechanism poc

### PR DESCRIPTION
**Pull request by nbittich**
Goal of this POC is to deploy a working example on dev. I tested it locally but I'd really appreciate a second look before deploying it; design is different than on figma, this is normal (will be finalize after demo it to Saska & Sofie)

### CODE REVIEW

File: app/templates/administrative-units/administrative-unit/ministers/new.hbs [hunk 1]
```diff
@@ -96,12 +97,19 @@
         </:card>
       </EditCard>
 
-      <Position::ContactEditCard
+      {{!-- <Position::ContactEditCard
```
Comment (claire-lovisa) [inline app/templates/administrative-units/administrative-unit/ministers/new.hbs:100]: Is the comment still needed for later ?

File: app/styles/_shame.scss [hunk 2]
```diff
@@ -135,3 +135,6 @@ table.dataTable tbody th {
 .items-center {
   align-items: center !important;
 }
+.zindex9998 {
```
Comment (claire-lovisa) [inline app/styles/_shame.scss:138]: I'm not seeing it used in the PR, is it needed ?

**Review by claire-lovisa — COMMENTED** [review]
It's looking very good !! I wrote down some remarks while testing it, some might not be valid, but here's my thoughts : 
- When I go to a persons's position, select an address and save, I'm redirected to a bestuursorgaan without any mandatarissen / bestuursleden (but I should, because I just updated a position ?)
- The addresses showing in the list, for a person, are the active addresses. Technically, I totally get why it's the case, but jut to be sure, did we expect that older addresses added for that same person would show up ?
- Doing the following when there is already a contact detail in the list will replace it by blank contact details : "Nieuw" -> Don't fill anything -> "Opslaan" (+ it's not persisted, when I refresh the old details are back)
- Suggestion - Reuse the buttons formatting of "Bewerk kerngegevens" to make it stand out a little more ?

**Comment by nbittich** [comment]
Thanks :)

> * When I go to a persons's position, select an address and save, I'm redirected to a bestuursorgaan without any mandatarissen / bestuursleden (but I should, because I just updated a position ?)

It seems to be the same behavior on dev / acc. I don't think I changed anything there 🤔  The address is persisted tho, if you go back to the position, it is correctly selected.


>     * The addresses showing in the list, for a person, are the active addresses. Technically, I totally get why it's the case, but jut to be sure, did we expect that older addresses added for that same person would show up ?

Yes this is required in the FP. only active position's addresses should be listed.

>     * Doing the following when there is already a contact detail in the list will replace it by blank contact details : "Nieuw" -> Don't fill anything -> "Opslaan" (+ it's not persisted, when I refresh the old details are back)

Yes this is normal. All fields in a contact details are optionals + you must click on "Opslaan" (not the one from the modal but the one from the page itself) to have it persisted. If you press the second opslaan, it will be saved and all fields will be empty.
It doesn't seem to be a logical behavior, but I'd prefer to keep it as is for the purpose of the poc, I'll demo it as is to Sofie and see what she thinks of what should we do if an user clicks on new & don't fill anything. Should be fairly easy to adjust for the final implementation.
 
>     * Suggestion - Reuse the buttons formatting of "Bewerk kerngegevens" to make it stand out a little more ?

Design will probably change a lot in the final implementation when I got the feedback of Saska & Sofie 😄 

I've fixed the other comments, thank you !

**Review by claire-lovisa — APPROVED** [review]

**Pull request merged**